_includes/search.html

```diff
@@ -0,0 +1,16 @@
+<!-- borrowed heavily from https://github.com/18F/identity-site/blob/main/_includes/search.html -->
+<form
+  accept-charset="UTF-8"
+  action="https://search.usa.gov/search"
+  class="usa-search{% if include.size %} usa-search--{{ include.size }}{% endif %}"
+  id="search_form"
+  method="get"
+  role="search"
+>
+  <input name="utf8" type="hidden" value="&#x2713;" />
+  <input type="hidden" name="affiliate" id="affiliate" value="developers-login-gov" autocomplete="off" />
+  <input type="search" name="query" id="search-field-{{ include.id }}" autocomplete="off" class="usa-input" />
```

Comment:
Should we have a `<label>` here to accessibly label the input? Or `aria-label`? Not sure if the labelled section parent is meant to take the place of a field-specific label in this case. Otherwise, the `id` value we pass into this partial wouldn't really be necessary.